nservicebus/messaging/evolving-contracts.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Evolving messages contracts
+reviewed: 
+component: Core
+---
+
+In message-based systems the messages are part of the contract, which defines how services communicate with each other.
+
+Evolving contracts over time is not an easy task and the appropriate strategy should be defined individually for each system. Generally that problem can't be resolved on the infrastructure level, therefore NServiceBus users need to analyze their systems, consider how they evolve and define the strategy which will make the most sense in their particular circumstances.
+
+This article presents basic guidelines for choosing contracts evolution strategy, avoiding common mistakes and ensuring the contracts will be easy to evolve over time.
+
+
+## Designing contracts
+
+
+### Messages
+
+Messages should be:
+
+- **POCO classes**.
```

Comment:
link https://en.wikipedia.org/wiki/Plain_Old_CLR_Object and spell out the acronym